tracks/java/exercises/secret-handshake/mentoring.md

```diff
@@ -1,73 +1,80 @@
+### Points to consider in this task
+* binary numeral system understanding
```

Comment:
```suggestion

* binary numeral system understanding
```

Add a newline.

Reply:
But you probably want to list all these points under `### Reasonable solutions` and `### Talking points`

Reasonable solutions use x y z.
Talking points: talk about using x instead of y; what are z q x